### PR TITLE
fix(admin): Change general config icon to gear

### PR DESCRIPTION
**Pull request by edbourque0**
From my point of view, the box icon was confusing because I thought that the icon was not shown so I changed it to a settings gear.

Also added yarn.lock to gitignore and added prettier as a dev dependency.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "deploy:dev": "docker buildx build --push --tag stonith404/pingvin-share:development --platform linux/amd64,linux/arm64 ."
   },
   "devDependencies": {
-    "conventional-changelog-cli": "^3.0.0"
+    "conventional-changelog-cli": "^3.0.0",
+    "prettier": "3.3.3"
```
Comment (stonith404) [inline package.json:14]:
```suggestion
    "conventional-changelog-cli": "^3.0.0"
```
Is there a reason why you've added this dependency?

**Comment by stonith404** [comment]
Thank you, yes I think the new icon makes more sense. Please see my other comment.

File: package.json (hunk 1 above)
Comment (edbourque0) [reply package.json:14]: I have vary limited knowledge of javascript but it asked me to install prettier to run the format script so I added it to the dependencies to resolve the install problem.
(I have no idea if this makes sense or not, sorry if it's not 😅)

Comment (stonith404) [reply package.json:14]: Okay, no problem. :) Then, I’ll simply undo the change.

**Pull request merged**